pom.xml

```diff
@@ -101,6 +101,7 @@
         <module>presto-main</module>
         <module>presto-ml</module>
         <module>presto-geospatial</module>
+        <module>presto-geospatial-toolkit</module>
```

Comment:
@mbasmanova FYI, github doesn't attribute this commit to you.

Reply:
@findepi , I don't think I understand this. I see my name next to 2 commits in https://github.com/prestodb/presto/pull/9475/commits What am I missing?

Reply:
I also see your _name_, but usually Github displays _login_, linking to your account. See also question mark next to your name at https://github.com/prestodb/presto/pull/9475/commits/75ea69f82535b03509a3b82d20a438d3fde82005

Reply:
Hmm... @findepi, I don't see questing mark. Here is what I see:

<img width="516" alt="screen shot 2017-12-05 at 12 05 29 pm" src="https://user-images.githubusercontent.com/27965151/33620028-b7085c1c-d9b4-11e7-865d-7aef1833d714.png">